app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,20 @@
+class OrderItemsController < ApplicationController
+  def create
+    item = set_quantity
+    item.save
+    redirect_back(fallback_location: root_path)
+  end
+
+  def destroy
+    OrderItem.delete params[:id]
+    redirect_back(fallback_location: root_path)
+  end
+
+  private
+
+  def set_quantity
```

Comment:
You have order_item create/update logic in one method and save it in controller action. Just save item in this method and call it in controller action, since you don't use this item anywhere